frontend/src/views/Apps/AppPassphrase/AppPassphrase.js

```diff
@@ -319,8 +319,9 @@ class AppPassphrase extends Component {
         <Row className="mb-5 app-networks">
           <Col>
             <Segment label="Networks">
-              <BootstrapTable
-                classes="table app-table app-table-empty table-striped"
+              <AppTable
+                scroll
+                toggler={[].length > 0}
```

Comment:
Missing **app-table-empty** css class. Why?

Reply:
I encapsulated all of that logic on the `<AppTable>` component

Reply:
Good